cmd/preflight/cmd/check_container.go

```diff
@@ -237,9 +243,16 @@ func checkContainerPositionalArgs(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+// validateConditions run all pre-run functions
+func validateConditions(cmd *cobra.Command, args []string) error {
+	err := validateCertificationProjectID()
+	checkForNewerReleaseVersion(cmd)
+	return err
+}
+
```

Comment:
I concur here. It seems odd here to call another function between setting err and returning it. Not sure about the use of MatchAll. But it wouldn't hurt to try it. Otherwise, at the very least I would suggest something like

```suggestion
// validateConditions run all pre-run functions
func validateConditions(cmd *cobra.Command, args []string) error {
	checkForNewerReleaseVersion(cmd)
	return validateCertificationProjectID()
}

```